app/scripts/lib/validate.js

```diff
@@ -146,6 +151,15 @@ define(function (require, exports, module) {
       return _.all(_.keys(data), function (key) {
         return isDataValid(data[key], schema[key]);
       });
+    },
+
+    /**
+     * Check whether string is a url. Validates url with or without http(s) and trailing slash.
+     *
+     * http://blog.mattheworiordan.com/post/13174566389/url-regular-expression-for-links-with-or-without
+     */
+    isUrlValid: function isUrlValid (url){
+      return urlRegEx.test(url);
```

Comment:
@rfk the relier in vlad's link is created [here](https://github.com/mozilla/fxa-content-server/blob/7d8c2c0661c4492895d39cc10ebcbd602f4bf596/app/tests/spec/views/ready.js#L32), appears to be Sync based and uses redirectUri. Is this correct? Should it be using an `urn` as the `redirectUri`

@shane-tomlinson I looked but couldn't find in project.